openconfig_pyang/plugins/openconfig.py

```diff
@@ -711,7 +711,7 @@ def check_enumeration_style(ctx, stmt):
       if re.match(r"[a-z]", enum.arg):
         err_add(ctx.errors, stmt.pos, "OC_ENUM_CASE",
                 (enum.arg, enum.arg.upper()))
-      elif not re.match(r"^[A-Z0-9][A-Z0-9\_\.]+$", enum.arg):
+      elif not re.match(r"^[A-Z0-9]?[A-Z0-9\_\.]+$", enum.arg):
```

Comment:
Shall we make the second part of this optional rather than the first -- and update the style guide? As you say:

```yang
  leaf a {
    type enumeration {
      enum _;
      enum *;
      enum .;
      enum ^;
    }
  }
```

seems to be valid, but quite frankly makes me feel ill :-)

Reply:
Agreed & done.